app/commands/google_service.py

```diff
@@ -25,25 +27,10 @@ def open_modal(client, body):
     client.views_open(trigger_id=body["trigger_id"], view=view)
 
 
-def google_service_command(client, body):
-    open_modal(client, body)
-
-
-def list_folders():
-    service = get_google_service("drive", "v3")
-    results = (
-        service.files()
-        .list(
-            pageSize=25,
-            supportsAllDrives=True,
-            includeItemsFromAllDrives=True,
-            corpora="drive",
-            q="parents in '{}' and mimeType = 'application/vnd.google-apps.folder' and trashed=false and not name contains '{}'".format(
-                SRE_INCIDENT_FOLDER, "Templates"
-            ),
-            driveId=SRE_DRIVE_ID,
-            fields="nextPageToken, files(id, name)",
-        )
-        .execute()
-    )
-    return results.get("files", [])
+def google_service_command(client, body, respond):
```

Comment:
Super nitpicky, but I would put some spacing between the end of the first function and the def googe_service_command

Reply:
I think this is showing without space due to the way GitHub displays the changes inline 🤔 

It shows like this in VSCode:
![image](https://github.com/cds-snc/sre-bot/assets/1690085/2f37b457-6338-4b72-8f99-2fa804232617)